src/services/FulfillmentService.js

```diff
@@ -45,12 +45,24 @@ class FulfillmentService {
     try {
       const {
         params: { slug },
-        body: { type, value },
+        body: { location },
       } = req;
 
-      // TODO: get items by slug,type and value
-
-      return referrals;
+      const requestParams = {
+        type: slug.split('/')[1] || '',
+        county: location || '',
+      };
+      const searchValue = Object.values(requestParams).join('/');
```

Comment:
This seems cryptic to me, relying on the order of fields defined in the requestParams object. I would rather see it be a little more straightforward.

```
const type = slug.split('/')[1] || '';
const county = location || '';
const url = `${process.env.PROJECT_URL}/rest/v1/fulfillments/referrals/${type}/${county}`;
```

Reply:
is it possible that user will not provide some of this params ?
for example we will get only location or only type ?